### PR TITLE
Change names of tests to align with endpoint

### PR DESCRIPTION
**Pull request by jcox-dev**
Some copy-pasta that was overlooked more than likely.

### CODE REVIEW

File: src/test/java/uk/gov/ons/ctp/response/notify/message/ActionInstructionReceiverImplTest.java [hunk 1]
```diff
@@ -116,6 +116,7 @@ public void testProcessBRESInstructionNotifyThrowsException() throws Notificatio
       actionInstructionReceiver.processInstruction(
           ObjectBuilder.buildActionInstruction(
               "9a5f2be5-f944-41f9-982c-3517cfcfef3c", "Joe,Blogg,07742994131,tester@gmail.com", true));
+      // Fail the test if an exception is not thrown
```
Comment (benjefferies) [inline src/test/java/uk/gov/ons/ctp/response/notify/message/ActionInstructionReceiverImplTest.java]: junit 5 looks to have a nicer way to handle expecting exceptions https://stackoverflow.com/a/40268447/4695665

File: src/test/java/uk/gov/ons/ctp/response/notify/endpoint/StatusEndpointTest.java [hunk 2]
```diff
@@ -73,7 +73,27 @@ public void setUp() throws Exception {
...
     }
 
-       /**
+    /**
+     * Scenario where null is returned from the database
+     *
+     * @throws Exception when getJson does
+     */
+    @Test
+    public void getStatusMessageNull() throws Exception {
```
Comment (benjefferies) [inline src/test/java/uk/gov/ons/ctp/response/notify/endpoint/StatusEndpointTest.java]: I wonder if NotFound would be a better description as it looks like that's the expectation?

Comment (jcox-dev) [reply src/test/java/uk/gov/ons/ctp/response/notify/endpoint/StatusEndpointTest.java]: In that case, it's probably just a duplication of the test that follows and is not necessary to reimplement.

File: src/test/java/uk/gov/ons/ctp/response/notify/message/ActionInstructionReceiverImplTest.java (hunk 1 above)
Comment (jcox-dev) [reply src/test/java/uk/gov/ons/ctp/response/notify/message/ActionInstructionReceiverImplTest.java]: Do we have the version of Junit pinned somewhere?

Comment (benjefferies) [reply src/test/java/uk/gov/ons/ctp/response/notify/message/ActionInstructionReceiverImplTest.java]: I can't see where it's defined. Maybe surefire is controlling this as it looks like surefire uses 4.12 and that's what's included

**Pull request merged**